src/Symfony/Component/Serializer/Tests/Encoder/XmlEncoderTest.php

```diff
@@ -50,6 +50,26 @@ public function testEncodeScalar()
     /**
      * @group legacy
      */
```

Comment:
You will need to make sure that the `@group` annotation is kept at the `testSetRootNodeName()` method (see the failing build).